packages/contracts/contracts/ionic/levered/LeveredPositionFactoryFirstExtension.sol

```diff
@@ -109,7 +109,7 @@ contract LeveredPositionFactoryFirstExtension is
   }
 
   function getPositionsExtension(bytes4 msgSig) external view returns (address) {
-    // TODO
+    // TODO return the address of some freshly created levered position
```

Comment:
Does this TODO need to be handled right now?

Reply:
No, after we deploy and create a new position contract

Reply:
hamed